fastai/vision/augment.py

```diff
@@ -21,16 +21,28 @@
 class RandTransform(DisplayedTransform):
     "A transform that before_call its state at each `__call__`"
     do,nm,supports,split_idx = True,None,[],0
-    def __init__(self, p=1., nm=None, before_call=None, **kwargs):
+    def __init__(self,
+        p:float=1., # Probability of applying Transform
+        nm:str=None,
+        before_call=None, # Optional batchwise preprocessing function
+        **kwargs
+    ):
         store_attr('p')
         super().__init__(**kwargs)
         self.before_call = ifnone(before_call,self.before_call)
 
-    def before_call(self, b, split_idx):
+    def before_call(self,
+        b,
+        split_idx:int, # Index of the train/valid dataset
+    ):
         "Set `self.do` based on `self.p`"
```

Comment:
Please add Something like "This can be function overridden. Set `self.do` based on `self.p`"

This was brought to my attention by Zach

Reply:
Add it. I totally agree with this. In my first iteration, I was thinking that it's some kind of `set_up function` that we can preprocess some attributes like self.do, self.p, self.split_idx, ...